demos/gesture_recognition_demo/cpp_gapi/src/tracker.cpp

```diff
@@ -0,0 +1,359 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "tracker.hpp"
+#include <unordered_map>
+#include <algorithm>
+#include <utility>
+#include <limits>
+#include <memory>
+#include <vector>
+#include <tuple>
+#include <set>
+
+cv::Point Center(const cv::Rect &rect) {
+    return cv::Point(static_cast<int>(rect.x + rect.width * 0.5),
+                     static_cast<int>(rect.y + rect.height * 0.5));
+}
+
+TrackerParams::TrackerParams()
+    : min_track_duration(25),
+      forget_delay(150),
+      affinity_thr(0.85f),
+      shape_affinity_w(0.5f),
+      motion_affinity_w(0.2f),
+      min_det_conf(0.0f),
+      bbox_aspect_ratios_range(0.666f, 5.0f),
+      bbox_heights_range(1, 1280),
+      drop_forgotten_tracks(true),
+      max_num_objects_in_track(300),
+      averaging_window_size_for_rects(1),
+      averaging_window_size_for_labels(1) {}
```

Comment:
how this values affect quality or some other stuff? Do we need to calibrate them later?
How's about to put some comment here ?